apps/wallet/src/ui/app/pages/home/index.tsx

```diff
@@ -28,14 +20,8 @@ const HomePage = ({ disableNavigation, limitToPopUpSize = true }: Props) => {
     const initChecking = useInitializedGuard(true);
     const lockedChecking = useLockedGuard(false);
     const guardChecking = initChecking || lockedChecking;
-    const [visibility, setVisibility] = useState(
-        document?.visibilityState || null
-    );
-    const dispatch = useAppDispatch();
-    const activeAddress = useActiveAddress();
-    const network = useAppSelector(
-        ({ app: { apiEnv, customRPC } }) => `${apiEnv}_${customRPC}`
-    );
+    const [, setVisibility] = useState(document?.visibilityState || null);
+
     useEffect(() => {
```

Comment:
Is this necessary now? I think we can remove it

Reply:
done